cfgov/v1/preprocessed/css/molecules/image-text-50-50.less

```diff
@@ -0,0 +1,48 @@
+/* ==========================================================================
+   cfgov-refresh
+   Image and Text 50-50 molecule
+   ========================================================================== */
+
+.m-image-text-50-50 {
+    position: relative;
+    padding-bottom: 60px;
```

Comment:
That's not a bad idea... would save a lot of writing for the most common `unit()` use case. It could take an optional second param for overriding the default `@base-font-size-px`.